src/main/java/org/apache/mesos/hdfs/Scheduler.java

```diff
@@ -545,43 +545,39 @@ private boolean offerNotEnoughResources(Offer offer, double cpus, int mem) {
   }
 
   private void reconcileTasks(SchedulerDriver driver) {
-    updateReconciliationDeadline();
```

Comment:
Yes, but the scheduler can get many reregistered callbacks in its lifetime, meaning many potential reconciliations, and the boolean must be reset to false at the beginning of each reconciliation.